ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessRepository.scala

```diff
@@ -106,7 +106,7 @@ class DBProcessRepository(val dbConfig: DbConfig, val modelVersion: ProcessingTy
     }
   }
 
-  private def updateProcessInternal(processId: ProcessId, processDeploymentData: ProcessDeploymentData)(implicit loggedUser: LoggedUser): DB[XError[ProcessUpdated]] = {
+  private def updateProcessInternal(processId: ProcessId, processDeploymentData: ProcessDeploymentData, forceIncreaseVersion: Boolean)(implicit loggedUser: LoggedUser): DB[XError[ProcessUpdated]] = {
```

Comment:
It doesn't look like this argument cause that version increase is always forced. It looks rather like `increaseVersionWhenJsonChanged`?

Reply:
Maybe `increaseVersionWhenJsonNotChanged` ?

Reply:
Ah, u've right - should be negation there! LGTM.